vllm/core/block_manager_v2.py

```diff
@@ -194,17 +193,23 @@ def get_block_table(self, seq: Sequence) -> List[int]:
         assert all(b is not None for b in block_ids)
         return block_ids
 
-    def access_all_blocks_in_seq(self, seq, now):
-        # TODO add prefix caching support.
-        # Tracked here https://github.com/vllm-project/vllm/issues/3667
-        pass
+    def access_all_blocks_in_seq(self, seq: Sequence, now: float):
+        if self.enable_caching:
+            # Update the last accessed time of all the blocks accessed
+            # in this step.
+            block_table = self.block_tables[seq.seq_id]
+            block_ids = []
+            for block in block_table._blocks:
```

Comment:
we should avoid using protected APIs, e.g. `_blocks`. can we use this instead?
https://github.com/vllm-project/vllm/blob/ceaf4ed0030c7816537359b8efc750474149ce0f/vllm/core/block/block_table.py#L182

Reply:
Got it, would apply~